docs/src/test-api/class-test.md

```diff
@@ -1096,6 +1096,27 @@ test('less readable', async ({ page }) => {
 });
 ```
 
+You can use `test.fail.only` to focus on a specific test that is expected to fail. This is particularly useful when debugging a failing test or working on a specific issue.
```

Comment:
Remove this section - I would omit this as implied.

Reply:
Thanks for the feedback. Removed! :)